bugrap/src/main/java/org/vaadin/bugrap/BaseModel.java

```diff
@@ -0,0 +1,12 @@
+package org.vaadin.bugrap;
+
+import org.vaadin.bugrap.domain.BugrapRepository;
+
+public class BaseModel {
+
+	private static final BugrapRepository _repository = new BugrapRepository(DatabaseHelper.DB_LOCATION);
```

Comment:
It would be better to use simple camel case notation in variable names, without special chars like '_'.